### PR TITLE
New Exercises

### PR DESCRIPTION
**Pull request by alirezatabatabaeian**
Hello dear @ErikSchierboom 
I've added three new exercises(Bank account + Error handling + Forth).
Also, I've modified two exercises(Hello world + Change).
I hope you can merge it.

Thanks
Alireza

### CODE REVIEW

File: exercises/practice/hello-world/.meta/example.fc
Comment (ErikSchierboom) [inline exercises/practice/hello-world/.meta/example.fc:7]: `slice` seems useful!

File: exercises/practice/hello-world/wrappers/Template.ts
Comment (ErikSchierboom) [inline exercises/practice/hello-world/wrappers/Template.ts:32]: Lovely!

**Review by ErikSchierboom — APPROVED** [review]
LGTM

File: exercises/practice/hello-world/wrappers/Template.ts
Comment (alirezatabatabaeian) [reply exercises/practice/hello-world/wrappers/Template.ts:32]: I hope you can review the left tasks

**Comment by ErikSchierboom** [comment]
> I hope you can also review the new tasks

I did, and they look good! Hence my approval (so feel free to merge).

**Comment by ErikSchierboom** [comment]
> I guess I don't have any merge access!

You should have a Github invite to join the func team! Once joined, you'll have merge permissions.

**Pull request merged**
